app/src/main/java/com/example/moodtrackerproject/ui/notes/NotesViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.moodtrackerproject.ui.notes
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.moodtrackerproject.data.DataBaseRepository
+import com.example.moodtrackerproject.data.NoteBody
+import com.example.moodtrackerproject.utils.Preference
+
+class NotesViewModel : ViewModel() {
+    var isChecked: Boolean = false
+
+    private val notesList: MutableList<NoteBody>? = if (Preference.getNotes() == null) mutableListOf() else Preference.getNotes()
```

Comment:
Pass emptyList instead of making it nullable

Reply:
![image](https://user-images.githubusercontent.com/87538962/148660762-6ebb7801-efc8-4b3d-8b42-1b28d5776d4f.png)

Reply:
![image](https://user-images.githubusercontent.com/87538962/148660762-6ebb7801-efc8-4b3d-8b42-1b28d5776d4f.png)

Reply:
saving screens for later